src/main/java/com/wiley/elements/custom/Selectable.java

```diff
@@ -0,0 +1,27 @@
+package com.wiley.elements.custom;
+
+import com.wiley.elements.TeasyElement;
+import com.wiley.elements.types.TeasyElementList;
+
+public interface Selectable {
+
+    void selectAll();
+
+    void selectByText(String text);
+
+    void selectByValue(String value);
+
+    void selectByIndex(int index);
+
+    void selectRandom();
+
+    void selectRandomByAnotherTextThan(String text);
```

Comment:
potentially there could be a lof of "random but not..." "random and...." conditions. What if we make 1 method but make it accepting some different customizable strategies. (lets discuss)